x-pack/plugins/monitoring/server/deprecations.ts

```diff
@@ -85,6 +85,6 @@ export const deprecations = ({
     // vice versa.
     // ^ These deprecations should only be shown if they are explicitly configured for monitoring -- we should not show Monitoring
     // deprecations for these settings if they are inherited from the Core elasticsearch settings.
-    // See the Core implementation: src/core/server/elasticsearch/elasticsearch_config.ts
+    // See the Core implementation: src/core/packages/core/server/elasticsearch/elasticsearch_config.ts
```

Comment:
Old path was still broken... let's minimize noise.
```suggestion
    // See the Core implementation: src/core/server/elasticsearch/elasticsearch_config.ts
```